translations/id.xtb

```diff
@@ -1,14 +1,14 @@
 <translationbundle lang="id">
-<translation id="1003362845472634045">Email login terkirim</translation>
+<translation id="1003362845472634045">Surel masuk terkirim</translation>
```

Comment:
I prefer use *Email* instead of *surel*

Reply:
@ekoeryanto yes, I prefer "Email" too but there is "Surel" in Indonesian vocabulary. WHat should we do?  
[sur.el](https://kbbi.kemdikbud.go.id/entri/surel)